components/centraldashboard/public/components/resources/kubeflow-icons.js

```diff
@@ -6,7 +6,20 @@ import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
 
 import {html, htmlLiteral} from '@polymer/polymer/lib/utils/html-tag.js';
-import bolt from '../../assets/icons/ic_bolt.svg';
+
+/**
+ * Converts an imported folder into a Object.entries() type of
+ *  [[key, val], ...] list.
+ * @param {object} context
+ * @return {[...[string, string]]}
+ */
+function createFileMap(context) {
```

Comment:
What do you think about the following? Drops the use of the lambdas and returns an object rather than a tuple for more clarity. I'd actually go a step further though and have this helper function actually build the `<g>` tag elements and rename it to something like `getIconDefinitions`.

```suggestion
function createFileMap(context) {
return context.keys().map((k) => {
        const name = k.split(/\/|\\/).slice(-1)[0].replace('.svg', '');
        const data = stripSVG(context(k).default || context(k));
        return {name, data};
    });
}
}
```

Reply:
Done, in favor of the suggestion!